webpack-demo/src/__mock__/addremove.test.js

```diff
@@ -0,0 +1,67 @@
+import Todos from '../todolist.js';
+
+document.body.innerHTML = `
+<div class="container">
+<h4 class="title">My Today's To do List</h4>
+<form>
+    <input class="description" type="text" placeholder="Add to your List..." autofocus />
+    <button class="add-btn">
+        <i class="fa-solid fa-arrows-rotate fa-icon"></i>
+    </button>
+</form>
+<div class="todos">
+</div>
+<div class="clear">
+    <a class="clear-btn">Clear All Completed Tasks</a>
+</div>
+</div>
+`;
```

Comment:
- [ ] Great job so far 👏. You are almost there.  While I was trying to run tests locally on my PC, I encountered some problems because running `npm test` was not working. I think this is due to the structure of your folders and files.  I realized that you have some duplicate files. Example: `package.json`, `package-lock.json`,etc. This causes the problem because I saw that you defined your script commands in the package.json file located in the webpack-demo folder while they are supposed to be in the package.json at the root level of the repository. Please rework the folder structure of your repository 😉 🏃